Package doesn't compile

Using this code on an M1 MacBook Air, with Zerolog v1.26.1:
```go
package log

import (
	"os"

	"github.com/UnnoTed/horizontal"
	"github.com/rs/zerolog"
	"github.com/rs/zerolog/log"
)

func InitializeLogger(verboseLoggingEnabled bool) {
	level := zerolog.InfoLevel
	if verboseLoggingEnabled {
		level = zerolog.TraceLevel
	}

	log.Logger = log.
		Output(horizontal.ConsoleWriter{Out: os.Stderr}).
		With().
		Logger().
		Level(level) // TODO implement custom formatters
}
```

I get the following error:
```sh
github.com/UnnoTed/horizontal
../vendor/github.com/UnnoTed/horizontal/horizontal.go:64:2: undefined: onResize
FAIL	github.com/instadevhq/apiforge/script [build failed]
FAIL
```

--- a/horizontal_darwin.go
+++ b/horizontal_darwin.go
@@ -1,0 +1,22 @@
+package horizontal
+
+import (
+	"os"
+	"os/signal"
+
+	"golang.org/x/sys/unix"
+)
+
+func onResize() {
+	ch := make(chan os.Signal, 1)
+	sig := unix.SIGWINCH
+	signal.Notify(ch, sig)
+	go func() {
+		for {
+			select {
+			case <-ch:
+				resizeSeparator()
+			}
+		}
+	}()
+}

